pkg/utils/mageutils/magefile.go

```diff
@@ -76,6 +76,13 @@ func newUpVersions(releaseType string) *upVersions {
 }
 
 // UpVersions command to up latest version of the repository to the next, including the beta and rc next versions.
+// Outputs:
+// actualVersion: represents the actual version of the repository (v1.0.0-beta).
```

Comment:
the actual version of beta should be `v1.0.0-beta.1` we don't launch versions outside the format `vX.Y.Z-beta.n`

Reply:
actualVersion refers to the latest release of the repository